app/reducer.js

```diff
@@ -108,11 +108,19 @@ export default function reduce(state, action) {
       knobs[20+] are reserved for effects
       */
       knobs: [
+<<<<<<< 7e5a49000860e02cb7f285ae6e05a0b6a344177d
         100, 100, 100, 100, 100,
         100, 100, 100, 100, 127,
         127, 100, 100, 100, 100,
         100, 100, 100, 100, 100,
         100, 100,
+=======
+      100, 100, 100, 100, 100,
+      100, 100, 100, 100, 127,
+      127, 100, 100, 100, 100,
+      100, 100, 100, 100, 100,
+      100, 100
+>>>>>>> configure, implement and confirm FETCH_PROFILE reducer
```

Comment:
It looks like you have left some merge comments in here. I'd just remove whatever happened here, the indentation we had previously was good and we'd want to keep that.